pygmo/docstrings.cpp

```diff
@@ -1804,6 +1803,40 @@ See also the docs of the C++ class :cpp:class:`pagmo::cec2006`.
 )";
 }
 
+std::string luksan_vlcek1_docstring()
+{
+    return R"(__init__(dim = 3)
+
+Implementation of Example 5.1 in the report from Luksan and Vlcek.
+
+The problem is also known as the Chanied Rosenbrock function with trigonometric-exponential constraints.
+
+Its formulation in pagmo can be written as:
+
+.. math::
+   \begin{array}{rl}
+   \mbox{find:}      & -5 \le \mathbf x_i \le 5, \forall i=1..n\\
+   \mbox{to minimize: } & \sum_{i=1}^{n-1}\left[100\left(x_i^2-x_{i+1}\right)^2 + \left(x_i-1\right)^2\right]\\
+   \mbox{subject to:} & 3x_{k+1}^3+2x_{k+2}-5+\sin(x_{k+1}-x_{k+2}})\sin(x_{k+1}+x_{k+2}})
+   +4x_{k+1}-x_k\exp(x_k-x_{k+1})-3 \le UB, \forall k=1..n-2 \\
+                      & 3x_{k+1}^3+2x_{k+2}-5+\sin(x_{k+1}-x_{k+2}})\sin(x_{k+1}+x_{k+2}})
+   +4x_{k+1}-x_k\exp(x_k-x_{k+1})-3 \ge LB, \forall k=1..n-2 \\
+   \end{array}
+
+See: Lukšan, L., and Jan Vlcek. "Sparse and partially separable test problems for unconstrained and equality
```

Comment:
Same here regarding the "š" character. We should check whether it shows up correctly in the final sphinx docs and use it consistently.